util/http.py

```diff
@@ -0,0 +1,24 @@
+"""Utilities for dealing with HTTP."""
+from __future__ import annotations
+
+from gzip import compress
+
+__all__ = ['gzip']
+
+
+def gzip(data: str | bytes, max_size: int = 512, encoding='utf-8') -> tuple[bytes, bool]:
```

Comment:
512 does seem reasonable for this, but I think looking at the typical sizes of EDDN messages might be useful before setting this in stone.

Every message is always going to have:

1. The overall `{...}` enclosure.
2. The `header`, with something like a 20 character uploaderID.
3. The `$schemaRef`
4. A bare minimum of `message:{...}` with at least `timestamp` key:value and at least one other key:value.

So, let's show our working in comments *somewhere*.  512 is probably OK for this util function, but show some working for the EDDN use-case at the call site.